src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/SlimefunItemHitListener.java

```diff
@@ -0,0 +1,49 @@
+package io.github.thebusybiscuit.slimefun4.implementation.listeners;
+
+import javax.annotation.Nonnull;
+
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.EntityDamageByEntityEvent;
+import org.bukkit.inventory.ItemStack;
+
+import io.github.thebusybiscuit.slimefun4.core.handlers.WeaponUseHandler;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import io.github.thebusybiscuit.slimefun4.implementation.items.weapons.VampireBlade;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
+
+/**
+ * This {@link Listener} is responsible for calling the {@link WeaponUseHandler}.
+ * 
+ * @author Mooy1
+ * 
+ * @see VampireBlade
```

Comment:
Might make more sense to link the WeaponUseHandler here instead of the VampireBlade